packages-exp/auth-exp/src/core/persistence/persistence_user_manager.ts

```diff
@@ -113,19 +113,53 @@ export class PersistenceUserManager {
       );
     }
 
-    const key = _persistenceKeyName(userKey, auth.config.apiKey, auth.name);
+    // Use the first persistence that supports a full read-write roundtrip (._isAvailable()).
+    let chosenPersistence: PersistenceInternal | null = null;
```

Comment:
Instead of setting this to `| null`, just set it to inMemory here as the default. `_getInstance()` has almost no overhead

Reply:
done